src/main/java/dev/dkvl/womutils/WomUtilsPlugin.java

```diff
@@ -205,6 +249,259 @@ public void onResponse(Call call, Response response)
 		});
 	}
 
+	private void sendPlayerRequest(Request request, String target, boolean add)
+	{
+		okHttpClient.newCall(request).enqueue(new Callback()
+		{
+			@Override
+			public void onFailure(Call call, IOException e)
+			{
+				log.warn("Error adding new member {}.", e.getMessage());
+			}
+
+			@Override
+			public void onResponse(Call call, Response response) {
+				try
+				{
+					buildErrorMessage(response, add);
+				}
+				catch (Exception e)
+				{
+					if (add)
+					{
+						groupMembers.add(target);
+					} else {
+						groupMembers.remove(target);
+					}
+				}
+				response.close();
+			}
+		});
+	}
+
+	private void buildErrorMessage(Response response, boolean add) throws IOException {
+		Type messageType = new TypeToken<JsonObject>() {}.getType();
+		String responseBody = response.body().string();
+		JsonObject data = gson.fromJson(responseBody, messageType);
+		String errorMessage = data.get("message").toString().replace("\"", "");
+
+		if (!add && errorMessage.contains("Successfully"))
+		{
+			return;
+		}
```

Comment:
This looks sketch. You should check the status code of the response instead of checking for `Successfully` in the message. Also that JsonObject deserialation-serialisation-replace should just be `WomError error = gson.fromJson(responseBody.string(), WomError.class);`. With a single field `WomError` class.